test/scripts/get-contour-upgrade-from-version.sh

```diff
@@ -6,7 +6,7 @@ set -o pipefail
 
 if CURRENT_TAG=$(git describe --tags --exact-match 2>/dev/null); then
   # We are on a tag, so find previous tag to this one.
-  git tag -l --sort=-v:refname | grep -A10 $CURRENT_TAG | grep -v 'alpha\|beta\|rc' | head -1
+  git tag -l --sort=-v:refname | grep -v 'alpha\|beta\|rc' | grep -A1 -x $CURRENT_TAG | grep -v "$CURRENT_TAG" | head -1
```

Comment:
```suggestion
  git tag -l --sort=-v:refname | grep -v 'alpha\|beta\|rc' | grep -A1 -x $CURRENT_TAG | tail -1
```

nit: might be able to simplify a little but otherwise 👍🏽

Reply:
ah yeah good point, let me make that change.